integration/test/clusterstate/main_test.go

```diff
@@ -104,13 +106,24 @@ func init() {
 // TestMain allows us to have common setup and teardown steps that are run
 // once for all the tests https://golang.org/pkg/testing/#hdr-Main.
 func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	var err error
+	var v int
+
 	{
-		c := setup.Config{
+		c := e2esetup.Config{
 			AWSClient: c,
 			Guest:     g,
 			Host:      h,
 		}
 
-		setup.Setup(m, c)
+		err = e2esetup.Setup(ctx, m, c)
+		if err != nil {
+			l.Log(fmt.Sprintf("%#v\n", err))
+			v = 1
+		}
 	}
+
+	os.Exit(v)
```

Comment:
Do we expect the setup to become more complex? I think we could remove the `v` magic and just `os.Exit(1)` where we have the error. Then we can also get rid of `var err error` and the whole thing is way leaner.

Reply:
No this is it. Good point I'll do that here and in the chart test.